packages/solo/src/init-basedir.js

```diff
@@ -53,7 +53,10 @@ export default function initBasedir(
   fs.copyFileSync(path.join(`${here}/..`, pj), path.join(dstHtmldir, pj));
   const gr = 'git-revision.txt';
   try {
-    fs.copyFileSync(path.join(`${here}/..`, gr), path.join(dstHtmldir, gr));
+    fs.copyFileSync(
+      path.join(`${here}/../public`, gr),
```

Comment:
Using / in path.join is odd.

Reply:
Good catch.  I'll fix it.

Reply:
Actually, not necessary.  Slashes are the canonical internal Node.js path separator, so:

On Windows:
```js
const path = require('path');
path.join('foo/bar/bingo/..', '../baz/bot') === 'foo\\baz\\bot'
// true
```

On MacOS:
```js
const path = require('path');
path.join('foo/bar/bingo/..', '../baz/bot') === 'foo/baz/bot'
// true
```

We also take advantage of this property in `packages/xsnap/test/test-boot-lockdown.js` and `packages/xsnap/test/test-snapstore.js`, so I don't think I'll change it after all.

Reply:
Waffling.  I'll make it clearer rather than template literals.

Reply:
> ... Slashes are the canonical internal Node.js path separator

TIL.